ios/FluentUI/Pill Button Bar/PillButtonTokens.swift

```diff
@@ -48,13 +48,13 @@ open class PillButtonTokens: ControlTokens {
     open var titleColor: PillButtonDynamicColors {
         switch style {
         case .primary:
-            return .init(rest: DynamicColor(light: globalTokens.neutralColors[.grey38], dark: globalTokens.neutralColors[.grey84]),
-                         selected: DynamicColor(light: globalTokens.neutralColors[.white], dark: globalTokens.neutralColors[.black]),
+            return .init(rest: DynamicColor(light: aliasTokens.foregroundColors[.neutral2].light, dark: aliasTokens.foregroundColors[.neutral2].dark),
```

Comment:
if the light and dark matches the same alias and it is intenteded, why not simplify rather than using dyanmic color?